scripts/travis-ci/before_install.bash

```diff
@@ -5,14 +5,45 @@
 # that can be found in the LICENSE file at the root of the
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
+MUMBLE_HOST_DEB=${MUMBLE_HOST/_/-}
+
 if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
-	if [ "${MUMBLE_QT}" == "qt4" ]; then
+	if [ "${MUMBLE_QT}" == "qt4" ] && [ "${MUMBLE_HOST}" == "x86_64-linux-gnu" ]; then
 		sudo apt-get -qq update
 		sudo apt-get build-dep -qq mumble
-	elif [ "${MUMBLE_QT}" == "qt5" ]; then
+	elif [ "${MUMBLE_QT}" == "qt5" ] && [ "${MUMBLE_HOST}" == "x86_64-linux-gnu" ]; then
 		sudo apt-get -qq update
 		sudo apt-get build-dep -qq mumble
 		sudo apt-get install qt5-default qttools5-dev qttools5-dev-tools qtbase5-dev qtbase5-dev-tools qttranslations5-l10n libqt5svg5-dev
+	elif [ "${MUMBLE_QT}" == "qt5" ] && [ "${MUMBLE_HOST}" == "i686-w64-mingw32" ]; then
+		sudo dpkg --add-architecture i386
+		sudo apt-get -qq update
+		echo "deb http://pkg.mxe.cc/repos/apt/debian wheezy main" | sudo tee /etc/apt/sources.list.d/mxeapt.list
```

Comment:
Well, I can try. But I think trusty is older than jessie, so the binaries might require a newer libc.